_includes/store/pcsg-recurring-budget-bundle.md

```diff
@@ -1,19 +1,23 @@
-Set up your projects to receive a fixed amount of budget on a regular basis (e.g. 05:00 hours every month). 
-Ideal for contracts with a regular budget bonus (e.g. maintenance agreements, SLAs etc.).
+A Kimai plugin that automatically adds a recurring money or time budget to your projects with a custom interval. Set
+up your projects to receive a fixed amount of budget on a regular basis (e.g. 05:00 hours every month). Ideal for
+contracts with a regular budget bonus (e.g. maintenance agreements, SLAs etc.).
+
+# Now ready for Kimai 2!
```

Comment:
Mentions of versions always confuse users. I have a sad history of that. I will remove it.
```suggestion
```